### PR TITLE
Update RedundantSyntax.md

### PR DESCRIPTION
**Pull request by kimnilsson**
Typo in RedundantSyntax.md

### CODE REVIEW

File: docs/rules/RedundantSyntax.md [hunk 1]
```diff
@@ -31,7 +31,5 @@ Example:
 + println("Bar")
 
 - println(raw"Baz")
-+ println("Foo")
-
-println(raw"Foo\nBar")
```
Comment (bjaglin) [inline docs/rules/RedundantSyntax.md:36]: The intent of that line was to show that this wouldn't get rewritten. Did you remove it on purpose?

**Review by bjaglin — COMMENTED** [review]
Thanks for taking the time to update the docs!

**Comment by kimnilsson** [comment]
> Thanks for taking the time to update the docs!

You've made it so easy with the `Edit` button so I thought why not :+1:

File: docs/rules/RedundantSyntax.md (hunk 1 above)
Comment (kimnilsson) [reply docs/rules/RedundantSyntax.md:36]: Yes, that was my intention as it broke the pattern of the preceding three examples (or well, that's how I read it at least). But perhaps it's a good idea to keep it but with a comment saying that it's not rewritten as `raw` is indeed used.

**Comment by bjaglin** [comment]
Thanks!

**Pull request merged**